src/packagedcode/models.py

```diff
@@ -94,14 +103,80 @@
   -- code in source or compiled form or both.
 """
 
-class Package(object):
+
+class EnhancedModel(Model):
+    def __init__(self, deserialize_mapping=None, strict=True, **kwargs):
```

Comment:
What are `deserialize_mapping=None, strict=True` about? 
Also the class should be rather called `BaseModel` IMH